Pod/Classes/PhxPush.m

```diff
@@ -11,37 +11,35 @@
 #import "PhxChannel_Private.h"
 #import "PhxSocket.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface PhxPush ()
 
 @property (nonatomic, weak) PhxChannel *channel;
-@property (nonatomic, retain) NSString *event;
-@property (nonatomic, retain) NSString *refEvent;
-@property (nonatomic, retain) NSDictionary *payload;
+@property (nonatomic) NSString *event;
+@property (nonatomic) NSString *refEvent;
+@property (nonatomic) NSDictionary *payload;
 
-@property (nonatomic, copy) After afterHook;
-@property (readwrite) int afterInterval;
-@property (nonatomic, retain) NSTimer *afterTimer;
+@property (nullable, nonatomic, copy) After afterHook;
+@property (nonatomic) NSTimeInterval afterInterval;
```

Comment:
this property has been made `nonatomic` on purpose, because it wasn't being used in any place where it needed to be `atomic`